pkg/draft/pack/pack_windows_test.go

```diff
@@ -0,0 +1,120 @@
+package pack
```

Comment:
do we need a conditional build flag for windows, or is Go smart enough about the `_windows` filename that it implies the windows build tag? I recall reading about that somewhere...

Reply:
Hi @bacongobbler ,
As I can understand if we try to build it on a windows machine, there is no need to specify window build tag.

> (example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

https://golang.org/pkg/go/build/